crates/core_arch/src/arm_shared/neon/mod.rs

```diff
@@ -4179,6 +5835,10 @@ pub unsafe fn vext_s64<const N: i32>(a: int64x1_t, _b: int64x1_t) -> int64x1_t {
 #[cfg_attr(all(test, target_arch = "arm"), assert_instr("nop", N = 0))]
 #[cfg_attr(all(test, target_arch = "aarch64"), assert_instr("nop", N = 0))]
 #[rustc_legacy_const_generics(2)]
+#[cfg_attr(
+    target_arch = "aarch64",
+    stable(feature = "neon_intrinsics", since = "1.59.0")
+)]
 pub unsafe fn vext_u64<const N: i32>(a: uint64x1_t, _b: uint64x1_t) -> uint64x1_t {
     if N != 0 {
         unreachable_unchecked()
```

Comment:
I just noticed this: we should be using a static assert here, not `unreachable_unchecked`. This applies to all `vext_{s,u,p,f}64`.